src/Appwrite/Utopia/Response/Model/Variable.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Variable extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('$id', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Variable ID.',
+                'default' => '',
+                'example' => '5e5ea5c16897e',
+            ])
+            ->addRule('$createdAt', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Variable creation date in Unix timestamp.',
```

Comment:
Will this remain a unix timestamp, or will this become datetime?

Reply:
@PineappleIOnic we need to update this PR to work with the new datetime PR.

Reply:
Yeah, Currently waiting on it to be stable before merging datetime into this PR

Reply:
Updated to datetime.